packages/editor/src/client/panel/peerEditPanel.tsx

```diff
@@ -191,16 +192,14 @@ export function PeerEditPanel({id, onClose, onSave}: ImageEditPanelProps) {
         </Panel>
         {!isLoadingPeerProfile && isValidURL && (
           <Panel header={t('peerList.panels.information')}>
-            <Panel
-              bordered={true}
-              style={{
-                height: '200px',
-                marginBottom: '20px',
-                backgroundSize: 'cover',
-                backgroundImage: `url(${
-                  profile?.logo?.previewURL ?? 'https://via.placeholder.com/240x240'
-                }`
+            <ChooseEditImage
+              disabled
+              imageStyle={{
```

Comment:
I think you can set them inside the component and check a few places where the component is used. I think it's needed everywhere.

Reply:
I did that at the beginning but I am not sure about it. I will go through other places as well to see it